### DISCUSSION TITLE
SteamVR keeps crashing...

### DISCUSSION THREAD

**Discussion by Massacher** [alvr-org/ALVR#1961] [Q&A]
I am new to VR. Its been only a bit over a week since I've been playing with it.

In the log file I have a warning: "[ERROR] Audio record error: Audio devices with more than 2 channels are not supported. Please turn off surround audio."
That isn't really an issue. I'm not looking to record audio so its a moot point. The issue is that it keeps on crashing. The following error is at the bottom of the log file: "[INFO] Server disconnected".

Would be helpful if it could tell me why it disconnected.
I haven't touched most settings just resolution and frame rate. I have it set to 120Hz and I know the headset can display that as I can set it in the headset itself.

So is there a log file that SteamVR creates that could tell me why its crashing?
Its getting really frustrating crashing every few minutes. I don't know if its because I keep going slightly outside my boundary. It glitches when that happens and actions I'm doing in a game don't seem to work properly. I'm playing HL Alyx and it looks so beautiful at 120Hz. I don't want to go back to 90Hz or less.

I seek the guidance of someone more knowledgeable than I please.

**Comment by zmerp** [comment]
ALVR is recording audio to send it to your headset. Try changing your default audio device in the taskbar tray, and make sure ALVR game audio is set to default.
